liblangutil/ErrorReporter.cpp

```diff
@@ -86,11 +86,11 @@ void ErrorReporter::error(Error::Type _type, SourceLocation const& _location, Se
 	m_errorList.push_back(err);
 }
 
-bool ErrorReporter::checkForExcessiveErrors(Error::Type _type)
+bool ErrorReporter::checkForExcessiveErrors(Error::Type _type, bool _advance)
```

Comment:
Please make that function private again and instead add a new function that accomplishes what you need: The information on whether the error count is above the max or not.

Reply:
Should be handled in  8781370